tensorflow_serving/model_servers/main.cc

```diff
@@ -276,6 +276,17 @@ void RunServer(int port, std::unique_ptr<ServerCore> core,
   builder.RegisterService(&model_service);
   builder.RegisterService(&prediction_service);
   builder.SetMaxMessageSize(tensorflow::kint32max);
+  const std::vector<string> channel_arguments =
+      tensorflow::str_util::Split(channel_arguments_str, ",");
+  for (const string& channel_argument : channel_arguments) {
+    const std::vector<string> key_val =
+        tensorflow::str_util::Split(channel_argument, "=");
+    try { // Attempt to add as int, otherwise assume string.
```

Comment:
Sorry, we don't use exceptions (try/catch) in our code-base. Please rewrite it using one of the functions in tensorflow/core/lib/strings/numbers.h that returns a bool instead of throwing an exception.